pyaerocom/io/mscw_ctm/reader.py

```diff
@@ -182,8 +183,9 @@ class _PrivateFields:
         filepaths: list[str] | None = None
         files: list[str] | None = None
         data_dir: str | None = None
+        file_pattern: re.Pattern
 
-    def __init__(self, data_id=None, data_dir=None, **kwargs):
+    def __init__(self, data_id: str | None = None, data_dir: str | None = None, **kwargs):
```

Comment:
documentation of file_pattern missing. Why not raise it to usual parameters and hide it in kwargs?